netcontrold/tests/lib/test_pmd_load_variance.py

```diff
@@ -13,46 +13,39 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 #
-
 import copy
-from unittest import mock
 from unittest import TestCase
-
-from netcontrold.app import ncd
+from unittest import mock
 from netcontrold.lib import dataif
 from netcontrold.lib import config
 
-_BASIC_DATA = """
-pmd thread numa_id 0 core_id 22:
-    emc hits:17461158
-    megaflow hits:0
-    avg. subtable lookups per hit:0.00
-    miss:0
-    lost:0
-    polling cycles:4948219259 (25.81%)
-    processing cycles:14220835107 (74.19%)
-    avg cycles per packet: 1097.81 (19169054366/17461158)
-    avg processing cycles per packet: 814.43 (14220835107/17461158)
---
-pmd thread numa_id 0 core_id 2:
-    emc hits:14874381
-    megaflow hits:0
-    avg. subtable lookups per hit:0.00
-    miss:0
-    lost:0
-    polling cycles:5460724802 (29.10%)
-    processing cycles:13305794333 (70.90%)
-    avg cycles per packet: 1261.67 (18766519135/14874381)
-    avg processing cycles per packet: 894.54 (13305794333/14874381)
-"""
+_FX_CPU_INFO = """
+processor       : 0
+core id         : 0
+physical id     : 0
 
+processor       : 1
+core id         : 1
+physical id     : 0
 
+processor       : 2
+core id         : 0
+physical id     : 0
+
+processor       : 3
+core id         : 1
+physical id     : 0
+"""
 
-class TestUtil_pmd_load_variance(TestCase):
+
+class TestRebalDryrunRR_OnePmd(TestCase):
```

Comment:
I think something you screwed up pulling old commit that I made .. it shows my additions.... I drop reviewing it and wait for new PR with fresh code changes.
Thanks.